File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/DateTimePicker/Functionality/ShowUpDownPropertyPage.cs
using System;
using System.Text;
using System.Data;
using System.Drawing;
using System.ComponentModel;
using System.Collections.Generic;

using Gizmox.WebGUI.Forms;
using Gizmox.WebGUI.Common;

namespace CompanionKit.Controls.DateTimePicker.Functionality
{
    public partial class ShowUpDownPropertyPage : UserControl
    {
        public ShowUpDownPropertyPage()
        {
            InitializeComponent();
        }

        /// <summary>
        /// Handles CheckedChanged event of the CheckBox that represents 
        /// a state of the DateTimePicker ShowUpDown property
        /// </summary>
        private void mobjUpDownButtonCheckBox_CheckedChanged(object sender, EventArgs e)
        {
            // Set value for the DateTimePicker ShowUpDown property according to checked state of the CheckBox
            this.mobjDemoDateTimePicker.ShowUpDown = this.mobjUpDownButtonCheckBox.Checked;
        }

        /// <summary>
        /// Handles Load event of the example' UserControl.
        /// </summary>
        private void ShowUpDownPropertyPage_Load(object sender, EventArgs e)
        {
            // Set initial checked state for the CheckBox that represents 
            // a state of the DateTimePicker ShowUpDown property
            this.mobjUpDownButtonCheckBox.Checked = this.mobjDemoDateTimePicker.ShowUpDown;
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/DateTimePicker/Functionality/ShowUpDownPropertyPage.vb
Namespace CompanionKit.Controls.DateTimePicker.Functionality

	Public Class ShowUpDownPropertyPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles CheckedChanged event of the CheckBox that represents
        ''' a state of the DateTimePicker ShowUpDown property
        ''' </summary>
        Private Sub mobjUpDownButtonCheckBox_CheckedChanged(sender As Object, e As EventArgs) Handles mobjUpDownButtonCheckBox.CheckedChanged
            ' Set value for the DateTimePicker ShowUpDown property according to checked state of the CheckBox
            Me.mobjDemoDateTimePicker.ShowUpDown = Me.mobjUpDownButtonCheckBox.Checked
        End Sub

        ''' <summary>
        ''' Handles Load event of the example' UserControl.
        ''' </summary>
        Private Sub ShowUpDownPropertyPage_Load(sender As Object, e As EventArgs) Handles Me.Load
            ' Set initial checked state for the CheckBox that represents
            ' a state of the DateTimePicker ShowUpDown property
            Me.mobjUpDownButtonCheckBox.Checked = Me.mobjDemoDateTimePicker.ShowUpDown
        End Sub
    End Class

End Namespace